foreman.te

```diff
@@ -269,6 +269,10 @@ corenet_tcp_connect_ssh_port(foreman_rails_t)
 corenet_udp_bind_generic_port(foreman_rails_t)
 corenet_udp_bind_generic_node(foreman_rails_t)
```

Comment:
Yes, I don't want to tackle them in this PR, I was quickly looking into this and probably this could be removed as a separate PR. But I fail to find those macros in sysnet_dns_name_resolve and I can't find any other suitable macro to replace these.

Reply:
Created https://projects.theforeman.org/issues/30849